policyengine_us/parameters/gov/states/va/tax/income/deductions/itemized/applicable_amount.yaml

```diff
@@ -9,6 +9,7 @@ JOINT:
   2019-01-01: 326_050 # 2020 no changes
```

Comment:
Do these amounts get updated with inflation? 
If so we will need to add some metadata to represent that,
See DC PR:
https://github.com/PolicyEngine/policyengine-us/pull/4306

Reply:
same for other files which seem to be updated yearly, we will need to check the legal code for that